Packages/com.unity.inputsystem/InputSystem/Controls/InputControlExtensions.cs

```diff
@@ -99,8 +100,14 @@ public static bool IsActuated(this InputControl control, float threshold = 0)
             var magnitude = control.EvaluateMagnitude();
             if (magnitude < 0)
             {
-                ////REVIEW: we probably want to do a value comparison on this path to compare it to the default value
-                return true;
+                // We know the control is not in default state but we also know it doesn't support
+                // magnitude. So, all we can say is that it is actuated. Not how much it is actuated.
+                //
+                // If we're looking for a specific threshold here, consider the control to always
+                // be under. But if not, consider it actuated "by virtue of not being in default state".
+                if (Mathf.Approximately(threshold, 0))
```

Comment:
Why not just `return Mathf.Approximately(threshold, 0) ?` instead of evaluating twice?